README.md

```diff
@@ -8,12 +10,12 @@ It is intented for decoding P12/PFX-stored certificate+key for use with the cryp
 ## Example
 
 ```go
-var p12, err = base64.StdEncoding.DecodeString(`base64-encoded-pfx-file`)
+p12, err := base64.StdEncoding.DecodeString(`base64-encoded-pfx-file`)

```

Comment:
This is just a matter of style that I've seen adopted in the Go community. Using var when declaring a variable with a zero value, otherwise using short initialization wherever possible. Plus it's more consistent with the code that follows.